packages/neon/neon/lib/src/blocs/accounts.dart

```diff
@@ -37,19 +37,28 @@ class AccountsBloc extends Bloc implements AccountsBlocEvents, AccountsBlocState
     });
 
     final as = accounts.value;
+
     if (_globalOptions.rememberLastUsedAccount.value && _storage.containsKey(_keyLastUsedAccount)) {
       final lastUsedAccountID = _storage.getString(_keyLastUsedAccount);
-      activeAccount.add(lastUsedAccountID != null ? as.find(lastUsedAccountID) : null);
-    } else {
-      unawaited(
-        _globalOptions.initialAccount.stream.first.then((final lastAccount) {
-          final account = lastAccount != null ? as.find(lastAccount) : null;
-          if (account != null) {
-            activeAccount.add(account);
-          }
-        }),
-      );
+      if (lastUsedAccountID != null) {
+        final aa = as.find(lastUsedAccountID);
+        if (aa != null) {
+          activeAccount.add(aa);
+        }
+      }
     }
+
+    _globalOptions.initialAccount.stream.listen((final lastAccount) {
```

Comment:
Why do we need to listen to this?
we only rely on the executed logic on startup but are still listening to it the for entire life cycle of the app.

Reply:
I forgot to revert it to the `.stream.first.then`